pages/admin/permit-holder/[permitHolderId].tsx

```diff
@@ -2,94 +2,162 @@ import { GetServerSideProps } from 'next'; // Get server side props
+import GuardianInformationCard from '@components/permit-holders/GuardianInformationCard'; // Guardian Information card
 import AppHistoryCard from '@components/permit-holders/AppHistoryCard'; // APP History card
 import AttachedFilesCard from '@components/permit-holders/AttachedFilesCard'; // Attached Files card
 import MedicalHistoryCard from '@components/permit-holders/MedicalHistoryCard'; // Medical History card
+import { GetPermitHolderRequest, GetPermitHolderResponse } from '@tools/pages/permit-holders/types';
+import { GET_PERMIT_HOLDER } from '@tools/pages/permit-holders/queries'; // Permit holder query
+import { useQuery } from '@apollo/client'; // Apollo
+import { useState } from 'react'; // React
 
-// TEMPORARY MOCK DATA
-const mockApplication = {
-  applicant: {
-    id: 1,
-    rcdUserId: 1,
-    mostRecentAPP: 12345,
-    firstName: 'Permit Holder',
-    lastName: 'One',
-    gender: Gender.Female,
-    dateOfBirth: 1,
-    email: 'applicantone@email.com',
-    phone: '1234567890',
-    province: Province.Bc,
-    city: 'Richmond',
-    addressLine1: '123 Richmond St.',
-    postalCode: 'X0X0X0',
-  },
-  physician: {
-    id: 1,
-    mspNumber: 123456789,
-    name: 'Physician One',
-    phone: '1234567890',
-    addressLine1: '123 Richmond St.',
-    postalCode: 'X0X0X0',
-    city: 'Richmond',
-    province: Province.Bc,
-    status: PhysicianStatus.Active,
-  },
-  guardian: {
-    id: 1,
-    firstName: 'Guardian',
-    lastName: 'One',
-    phone: '1234567890',
-    addressLine1: '123 Richmond St.',
-    postalCode: 'X0X0X0',
-    city: 'Richmond',
-    province: Province.Bc,
-    relationship: 'Parent',
-  },
-  createdAt: new Date().toDateString(),
-  expirationDate: new Date().toDateString(),
-  isRenewal: true,
-  applicationStatus: ApplicationStatus.Pending, // Will be updated in the future as we'll be able to reference ApplicationStatus enum in types.ts
-  permitFee: 5,
-  donation: 10,
-  paymentType: PaymentType.Visa,
+export type ApplicantData = {
+  id: number;
+  rcdUserId?: number;
+  firstName: string;
+  lastName: string;
+  gender: Gender;
+  dateOfBirth: Date;
+  email?: string;
+  phone: string;
+  province: Province;
+  city: string;
+  addressLine1: string;
+  addressLine2?: string;
+  postalCode: string;
+  status?: ApplicantStatus;
+};
+
+export type PermitData = {
+  rcdPermitId: number;
+  expiryDate: Date;
+  applicationId: number;
+  isRenewal: boolean;
+  status: ApplicationStatus;
+};
+
+export type PermitHolderAttachedFile = {
+  associatedApp?: number;
+  dateUploaded: Date;
+  fileUrl: string;
+};
+
+export type MedicalHistoryEntry = {
+  disablingCondition: string;
+  dateUploaded: Date;
+  associatedApplication: {
+    disability: string;
+    affectsMobility: boolean;
+    mobilityAidRequired: boolean;
+    cannotWalk100m: boolean;
+    aid: Aid[];
+    createdAt: Date;
+  };
+};
+
+export type PreviousPhysicianData = {
+  name: string;
+  mspNumber: number;
+  phone: string;
+};
```

Comment:
These types should go in a separate tools file

We should also take advantage of TypeScript's `Pick` to "pick" the fields we want from existing types in our `lib/graphql/types.ts` file rather than declaring new types which may fall out of sync with our GraphQL API. We want TypeScript to raise errors whenever any of these dependencies change.